src/scval.js

```diff
@@ -151,6 +152,10 @@ export function nativeToScVal(val, opts = {}) {
         return val.toScVal();
       }
 
+      if (val instanceof Keypair) {
```

Comment:
may be worthwhile to see a unit test assert this happens, and what is expected if `publicKey` were null/undefined?

Reply:
nice call, added! `publicKey` can't be undefined, though, only the `secretKey` can if you create it `fromPublicKey` instead of `fromSecret`.